frontend/src/reports/ui/ModeSelectorBoxMobile.tsx

```diff
@@ -26,8 +25,8 @@ export default function ModeSelectorBoxMobile(
 ) {
   return (
     <div className='mode-selector-box-mobile'>
-      <Card raised={true} className='m-2 p-2 md:hidden'>
-        <Grid container justifyContent='flex-start'>
+      <div className='m-2 rounded-sm bg-white p-2 shadow-raised md:hidden'>
+        <div className='flex justify-start'>
           <div>
```

Comment:
I think we can probably remove at least one if not two of these `<div>`s; it's a small thing so if it's difficult don't worry about it, but as part of this refactor im trying to make sure we can make the cleanest, tightest, most semantic html that we can.

Reply:
Ok! I'll take a look today